todoapp/src/components/Icons/AddIcon.vue

```diff
@@ -0,0 +1,29 @@
+<!-- eslint-disable vue/multi-word-component-names -->
```

Comment:
You can add the rule: vue/multi-word-component-names into the eslintrc file, this way you do not need to specify it on top of files.